scripts/common

```diff
@@ -53,11 +53,10 @@ unmount_image(){
 	local LOOP_DEV=$(losetup -j ${1} | cut -f1 -d':')
 	if [ -n "${LOOP_DEV}" ]; then
 		local MOUNTED_DIR=$(mount | grep $(basename ${LOOP_DEV}) | head -n 1 | cut -f 3 -d ' ')
-		if [ -n "${MOUNTED_DIR}" ]; then
+		if [ -n "${MOUNTED_DIR}" && "${MOUNTED_DIR}" != "/" ]; then
```

Comment:
`[: missing ']'`

I think this should be `[ -n "${MOUNTED_DIR}" ] && [ "${MOUNTED_DIR}" != "/" ];`